collect_app/src/main/java/org/odk/collect/android/utilities/ViewIds.java

```diff
@@ -1,47 +1,17 @@
 package org.odk.collect.android.utilities;
 
-import android.os.Build;
 import android.view.View;
 
-import java.util.concurrent.atomic.AtomicInteger;
-
 public class ViewIds {
```

Comment:
Now it doesn't make sense to keep this class. Could you get rid of it and use `View.generateViewId()` everywhere instead?

Reply:
Yes @grzesiek2010, I will do that!